lib/ansible/modules/cloud/google/gce_pd.py

```diff
@@ -127,7 +127,7 @@ def main():
             instance_name=dict(),
             mode=dict(default='READ_ONLY', choices=['READ_WRITE', 'READ_ONLY']),
             name=dict(required=True),
-            size_gb=dict(default=10),
+            size_gb=dict(default=10, type='int'),
```

Comment:
Should the default be updated to be `default='10'`? Otherwise, I imagine the module gives a warning. I assume the same holds true for vlan_id in lib/ansible/modules/network/illumos/dladm_vlan.py.